invenio_accounts/tasks.py

```diff
@@ -55,3 +57,18 @@ def clean_session_table():
     for session in sessions:
         delete_session(sid_s=session.sid_s)
     db.session.commit()
+
+
+@shared_task
+def delete_ips():
+    """Automatically remove user.last_login_ip older than 30 days."""
+    expiration_date = datetime.utcnow() - timedelta(days=30)
+
+    users = User.query.filter(User.last_login_ip.isnot(None),
```

Comment:
This method will load the users into the Python layer. Since we don't need the data in Python, we can execute a database UPDATE statement instead like this:  

```python
User.query.filter(
    User.last_login_ip.isnot(None), 
    User.last_login_at < expiration_date
).update({
    User.last_login_ip: None
})
User.query.filter(
    User.current_login_ip.isnot(None), 
    User.current_login_at < expiration_date
).update({
    User.current_login_ip: None
})
db.session.commit()
```